codelists/models.py

```diff
@@ -157,6 +168,12 @@ def latest_visible_version(self, user):
 
         return self.visible_versions(user).first()
 
+    def latest_published_version(self):
```

Comment:
I think "published" and "visible" mean slightly different things, so could you rename this to `latest_visible_version`?

Reply:
There's already a `latest_visible_version`, which shows the last visible version for the user, and defaults to the last version with status "published" if the user is anonymous.  This method is intended to return the last published version, even if there's a non-published version that would be visible for this user.